android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.sentry">
+    package="io.sentry.react">
     <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
this is already merged automatically by `sentry-android-core` btw but not related to this PR